plugins/brimcap/brimcap-plugin.ts

```diff
@@ -214,11 +217,78 @@ export default class BrimcapPlugin {
     )
   }
 
+  private setupLoader() {
```

Comment:
Do you like the word loader over importer? Loader in the front end context makes me thing of spinning wheels. Importer might be less over"loaded".  I don't feel too strongly about it though.

Reply:
I could go either way, I do see your point about the semantic collision there. Part of why I like `load()` is just that it has an obvious rollback verb, "unload", whereas "unimport", well, has a red squiggly under it as I type 😄 

Mind if I keep it as is for now but flag this for more discussion when we chat api design next? I anticipate there will be a lot more changes like this that we may want to make